docs/video-processing/functionalities/use-mapping-tool.md

```diff
@@ -1,23 +1,106 @@
 ---
 layout: default
 title: Use the mapping tool
-parent: /video-processing/functionalities
-grand_parent: /video-processing
+parent: /video-processing/functionalities.html
+grand_parent: /video-processing.html
 nav_order: 2
 ---
 
 # Use the mapping tool
```

Comment:
I am conviced that the mapping tool is one of the big advantages of our software over others on the market. However, people demand a simple non-mapping option e.g. for the simulated videos.